ref/src/sw_core.F90

```diff
@@ -172,34 +171,6 @@ subroutine c_sw(sw_corner, se_corner, nw_corner, ne_corner,       &
       call fill2_4corners(delp, pt, 1, sw_corner, se_corner, ne_corner, nw_corner)
     end if
 
-    if ( hydrostatic ) then
-#ifdef SW_DYNAMICS
-      do j = js-1, jep1
-        do i = is-1, ie+2
-          if ( ut(i, j) > 0. ) then
-            fx1(i, j) = delp(i-1, j)
-          else
-            fx1(i, j) = delp(i, j)
-          endif
-          fx1(i, j) = ut(i, j) * fx1(i, j)
-        enddo
-      enddo
-#else
-      do j = js-1, jep1
-        do i = is-1, ie+2
-          if ( ut(i, j) > 0. ) then
-            fx1(i, j) = delp(i-1, j)
-            fx(i, j) = pt(i-1, j)
-          else
-            fx1(i, j) = delp(i, j)
-            fx(i, j) = pt(i, j)
-          endif
-          fx1(i, j) = ut(i, j) * fx1(i, j)
-          fx(i, j) = fx1(i, j) * fx(i, j)
-        enddo
-      enddo
-#endif
-    else
       if (grid_type < 3) then
```

Comment:
Please also adjust the indentation of the `else` part of the branch that is now no longer inside an `if`.